what_to_watch/suggest/suggestion.py

```diff
@@ -1,8 +1,16 @@
 """
 Movie suggestion handling
 """
+from typing import List
+import sys
+import pandas as pd # type: ignore
 
-def suggest_movie() -> str:
+F_DIR = 'resources/imdb_data.csv'
+ARGS = sys.argv[1:]
+IMDB_DATA = pd.read_csv(F_DIR)
```

Comment:
I think it'd be better to include this and anything else to only be run when calling the module directly below, because otherwise it will called every time we import the module anywhere else.
```python
if __name__ == "__main__":
     # code goes here
```